src/Monolog/Handler/LogdnaHandler.php

```diff
@@ -108,6 +118,11 @@ protected function write(\Monolog\LogRecord $record): void
             'ip' => $this->ip,
             'tags' => $this->tags
         ];
+
+        if ($this->include_request_time) {
+            $query['now'] = $this->getCurrentTimeInMilliseconds();
```

Comment:
We could replace this with just `(string)floor(microtime(true) * 1000)`.

Reply:
done